### DISCUSSION TITLE
Please make sure your database server is running at `undefined.ec2-52-212-228-71.eu-west-1.compute.amazonaws.com`:`5432`.

### DISCUSSION THREAD

**Discussion by 2manoj1** [prisma/prisma#14597] [Q&A]
When trying to **Heroku** Postgresql. Its throwing runtime error.

<img width="655" alt="Screenshot 2022-08-02 at 5 50 32 AM" src="https://user-images.githubusercontent.com/15128569/182266084-07e5950f-9278-4828-9456-0b83f847d8b3.png">

**Note**: Local **Docker** Postgresql working fine.

Please help me. How to resolve this issue.

  "@prisma/client": "4.1.1",
  "prisma": "4.1.1",

**Comment by austincrim** [comment]
Hey @2manoj1 👋,

Could you share your connection string (with sensitive info redacted)? Your database server starts with `undefined` which is surprising and points to an issue with the connection string/environment variables.

**Reply by 2manoj1** [reply]
Hi @austincrim,
I copied heroku URI string and paste in `.env` as DATABASE_URL
Below format I check. My Local Docker postgreSQL no issue.

DATABASE_URL="postgres://\<UserNAme\>:\<Password\>@\<HOST\>:\<PORT\>/\<DB\>"

<img width="1427" alt="Screenshot 2022-08-03 at 10 34 05 AM" src="https://user-images.githubusercontent.com/15128569/182528527-04d14889-65ce-414e-81fb-e4ed353ad91b.png">

Note: After try all but no luck so I move to `data-proxy` way to solve the issue as workaround for now.

**Reply by janpio** [reply]
If the Dataproxy can connect with this connection, so should you directly - unless somehow traffic to that port for example is blocked for public IPs/hostnames from your computer.

**Reply by 2manoj1** [reply]
@janpio 
From vercel deployed version also same problem. 
- Create Heroku app
- Create Heroku Postgress Database and attach to your app
- Copy URI string from settings 
- Add DATABASE_URL in Environment
- Redeploy 

Are vercel also blocking? 
Same issue I face in vercel

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
